src/proposals/20201128/SparkEthereum_20241128.t.sol

```diff
@@ -0,0 +1,218 @@
+        // TODO: replace with deployed payload
+        payload = deployPayload();
+        // TODO: remove after Sky spell executes on mainnet
+        // mock Sky approving 100M liquidity to spark
+        vm.prank(Ethereum.PAUSE_PROXY);
+        DssAutoLineLike(AUTO_LINE).setIlk(ALLOCATOR_ILK, 100_000_000e45, 100_000_000e45, 1 hours);
+        DssAutoLineLike(AUTO_LINE).exec(ALLOCATOR_ILK);
+
+        base.selectFork();
+        // TODO: replace with deployed payload
+        payloadBase = deployPayloadBase();
+        mainnet.selectFork();
+    }
+
+    function testWBTCChanges() public {
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', pool);
+        ReserveConfig memory wbtcConfig         = _findReserveConfigBySymbol(allConfigsBefore, 'WBTC');
+
+        assertEq(wbtcConfig.liquidationThreshold, 65_00);
+
+        executePayload(payload);
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', pool);
+        wbtcConfig.liquidationThreshold        = 60_00;
+
+        _validateReserveConfig(wbtcConfig, allConfigsAfter);
+    }
+
+    function testcbBTCChanges() public {
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', pool);
+        ReserveConfig memory cbBTCConfig        = _findReserveConfigBySymbol(allConfigsBefore, 'cbBTC');
+
+        assertEq(cbBTCConfig.liquidationThreshold, 70_00);
+        assertEq(cbBTCConfig.ltv, 65_00);
+
+        executePayload(payload);
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', pool);
+        cbBTCConfig.liquidationThreshold       = 75_00;
+        cbBTCConfig.ltv                        = 74_00;
+
+        _validateReserveConfig(cbBTCConfig, allConfigsAfter);
+    }
+
+    function testExistingMorphoVault() public {
+        MarketParams memory sUSDeVault =  MarketParams({
+            loanToken:       Ethereum.DAI,
+            collateralToken: PT_SUSDE_27MAR2025,
+            oracle:          PT_27MAR2025_PRICE_FEED,
+            irm:             Ethereum.MORPHO_DEFAULT_IRM,
+            lltv:            0.915e18
+        });
+
+        _assertMorphoCap(sUSDeVault, 200_000_000e18);
+        executePayload(payload);
+        _assertMorphoCap(sUSDeVault, 200_000_000e18, 400_000_000e18);
+
+        skip(1 days);
+        IMetaMorpho(Ethereum.MORPHO_VAULT_DAI_1).acceptCap(sUSDeVault);
+        _assertMorphoCap(sUSDeVault, 400_000_000e18);
+    }
+
+    function testNewMorphoVault() public {
+        MarketParams memory USDeVault =  MarketParams({
+            loanToken:       Ethereum.DAI,
+            collateralToken: PT_USDE_27MAR2025,
+            oracle:          PT_27MAR2025_PRICE_FEED,
+            irm:             Ethereum.MORPHO_DEFAULT_IRM,
+            lltv:            0.915e18
+        });
+
+        _assertMorphoCap(USDeVault, 0);
+        executePayload(payload);
+        _assertMorphoCap(USDeVault, 0, 100_000_000e18);
+
```

Comment:
this assertion holds both before and after executing the payload, I don't mind adding it but would appeciate some context on why is it relevant or how could the spell change it

Reply:
Some comments are provided just to align coding style of spells. To make it more consistent. 
@hexonaut, I saw [this line](https://github.com/marsfoundation/spark-spells/blob/master/src/proposals/20241114/SparkEthereum_20241114.t.sol#L135) several times. Could you please clarify when it should be used?

Reply:
Sometimes these things are added just out of an abundance of caution. In this case I'm fine either way. Include it if you want, but don't think it's necessary because as @0xteddybear said it's not changing.

Reply:
added the assertions on a7b14f2